src/main/scala/scalafix/sbt/ScalafixTestkitPlugin.scala

```diff
@@ -27,6 +27,11 @@ object ScalafixTestkitPlugin extends AutoPlugin {
   }
   import autoImport._
 
+  override def buildSettings: Seq[Def.Setting[_]] =
+    List(
+      resolvers += Resolver.sonatypeRepo("snapshots")
```

Comment:
users of a SNAPSHOT testkit plugin must resolve from sonatype snapshots both scalafix-testkit for the test project (where the plugin is enabled) AND scalafix-core for the actual rule project, thus the need for impacting the entire build and not just the project settings.